src/main.rs

```diff
@@ -169,9 +170,11 @@ async fn create_context(
         .context("No .credentials.toml found. Call command auth first.")?;
     let credentials: Credentials = toml::from_str(&serialized_credentials)?;
     let sw_client = SwClient::new(credentials, in_flight_limit).await?;
-    // ToDo: lookup entities.json definitions
 
-    // ToDo: further schema verification
+    let api_schema = sw_client.entity_schema().await;
+    let entity = &schema.entity;
+
+    let _ = validate_paths_for_entity(entity, &schema.mappings, &api_schema?);
```

Comment:
You don't want to throw away that result of the function, as it might contain an error. Just propagate the possible error up to the parent caller function by adding a question mark.
```suggestion
    validate_paths_for_entity(entity, &schema.mappings, &api_schema?)?;
```